tests/restapi/test_restapi.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+import time
+import logging
+import requests
+import json
+
+from tests.common.helpers.assertions import pytest_assert
+from restapi_operations import Restapi
+
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.disable_loganalyzer
+]
+
+CLIENT_CERT = 'restapiclient.crt'
+CLIENT_KEY = 'restapiclient.key'
+
+restapi = Restapi(CLIENT_CERT, CLIENT_KEY)
```

Comment:
We need to run this test only if the DUT has restapi docker running. Can we ensure that?